Source/ui_unix/mainwindow.h

```diff
@@ -65,7 +67,8 @@ class MainWindow : public QMainWindow
 	CPS2VM* m_virtualMachine = nullptr;
 	bool m_deactivatePause = false;
 	bool m_pauseFocusLost = true;
-#ifdef HAS_LIBEVDEV
+	CFutureContinuationManager m_futureContinuationManager;
```

Comment:
Did that get there in a rebase? I thought I got rid of that (CFutureContinuationManager) in master.